core/src/main/java/org/apache/accumulo/fate/ZooStore.java

```diff
@@ -486,7 +488,8 @@ public String timeTopCreated(long tid) {
 
     try {
       Stat stat = zk.getZooKeeper().exists(getTXPath(tid), false);
-      return new Date(stat.getCtime()).toString();
+      DateTimeFormatter dtf = DateTimeFormatter.ofPattern("E yyyy MMM dd z hh:mm:ss");
```

Comment:
iso time like 2021-03-31T10:00:00Z would be easier for external parsing / sorting

Reply:
That's fair. I will add it back in my next commit.

Reply:
![image](https://user-images.githubusercontent.com/29436247/113007018-d8bc7d80-9143-11eb-8243-8f5fd19852fe.png)